core/lib/protobuf_config/src/lib.rs

```diff
@@ -26,7 +26,7 @@ mod snapshots_creator;
 mod witness_generator;
 
 pub mod proto;
-mod repr;
+pub mod repr;
```

Comment:
sorry, but making dal depend on protobuf_config is not acceptable.

Reply:
Extracted to `zksync_protobuf` (https://github.com/matter-labs/era-consensus/pull/66).